### ISSUE TITLE
About the dataset settings in pre-training and fine-tuning

### ISSUE THREAD

**Issue by jsadu826** [Luffy03/VoCo#16]
Hi, thank you for your productive work. I have some questions:

1. When pre-training, why split the dataset into training and validation, and use only the training set but not the entire dataset?
2. When fine-tuning on the BTCV, since BTCV's testing set does not contain labels, how are the DICE scores calculated in the paper?

**Comment by jsadu826** [comment]
Thank you!

I'm also working on a project of SSL pre-training on medical images, so my understanding is that for datasets whose online evaluation is no longer available, such as BTCV, they should not be considered as downstream tasks any more, but more up-to-date datasets/grand challenges should be considered.

**Comment by Luffy03** [comment]
> Thank you!
> 
> I'm also working on a project of SSL pre-training on medical images, so my understanding is that for datasets whose online evaluation is no longer available, such as BTCV, they should not be considered as downstream tasks any more, but more up-to-date datasets/grand challenges should be considered.

Yes, you are right! There are a lot of available leaderboards, e.g., amos, flare22, flare23, abdomen1k. More implementations and details will soon be uploaded. Stay tuned!
